ui/onboarding/Login.qml

```diff
@@ -189,6 +189,7 @@ Item {
             anchors.topMargin: 26
             anchors.horizontalCenter: parent.horizontalCenter
             font.pixelSize: 13
+            bgColor: Style.current.transparent
```

Comment:
I think the solution was just to use `type: "secondary"`

Reply:
Only using the "secondary" type is not enough. We need to consider using the new "warn" type button.
What do you think about this?

Reply:
I'm not sure I understand what you mean. `warn` is for a read button no?
We want the button to have no background by default and a background on hover, and that's what `secondary` does

Reply:
![image](https://user-images.githubusercontent.com/29419562/105734552-2e11be00-5f4c-11eb-8a21-703953922b8a.png)
![image](https://user-images.githubusercontent.com/29419562/105734570-34a03580-5f4c-11eb-844c-dd8a0100d47a.png)
I meant above transparent warn theme button.

Reply:
`secondary` does a transparent background by default too and the color of the button remains correct instead of being red.

Reply:
ok, let me follow you